tests/conftest.py

```diff
@@ -84,6 +84,7 @@ def app(test_path, redis_server):
             "CACHE_PATH": test_path,
             "TESTING": True,
             "UPSTREAM_URL": "http://localhost:8001",
+            "MAX_CUSTOM_ROOTFS_SIZE_MB": 100,
```

Comment:
Would it be fine if this was reverted then?

Reply:
Yea not needed